the-shore/patterns/home.php

```diff
@@ -0,0 +1,205 @@
+<div class="wp-block-column" style="flex-basis:66.66%"><!-- wp:paragraph -->
+<p><?php echo __('Wake up to stunning vistas of the turquoise Caribbean waters. Our rooms provide a tranquil retreat, complete with luxurious bedding, contemporary furnishings, and a private balcony to enjoy the mesmerizing ocean views.', 'the-shore');?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:group {"style":{"spacing":{"margin":{"top":"3rem","bottom":"3rem"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"stretch"}} -->
+<div class="wp-block-group" style="margin-top:3rem;margin-bottom:3rem"><!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_4.jpg" alt="<?php echo __('', 'the-shore');?>"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_5.jpg" alt="<?php echo __('', 'the-shore');?>"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:group --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"50px"} -->
+<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_6.jpg","dimRatio":20,"overlayColor":"black","isUserOverlayColor":true,"minHeight":600,"minHeightUnit":"px","style":{"spacing":{"padding":{"right":"6vw","left":"6vw"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-cover" style="padding-right:6vw;padding-left:6vw;min-height:600px"><span aria-hidden="true" class="wp-block-cover__background has-black-background-color has-background-dim-20 has-background-dim"></span><img class="wp-block-cover__image-background" alt="<?php echo __('', 'the-shore');?>" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_6.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase","letterSpacing":"0.1em"}},"fontSize":"x-small","fontFamily":"commissioner"} -->
+<p class="has-commissioner-font-family has-x-small-font-size" style="letter-spacing:0.1em;text-transform:uppercase"><?php echo __('Bars &amp; Restaurant', 'the-shore');?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"textAlign":"left","style":{"typography":{"fontStyle":"normal","fontWeight":"300"}},"fontSize":"large"} -->
+<h2 class="wp-block-heading has-text-align-left has-large-font-size" style="font-style:normal;font-weight:300"><?php echo __('Enjoy a cocktail at our stylish bar, or savor a fine dining experience with a backdrop of stunning ocean views.', 'the-shore');?></h2>
+<!-- /wp:heading -->
```

Comment:
While the visual styles of these lend themselves to the `Bars &amp; Restaurant` being a `<p>` and the `Enjoy a cocktail at our stylish bar, or savor a fine dining experience with a backdrop of stunning ocean views.` a larger `<h2>`, these tags should be switched.

`Bars &amp; Restaurant` is the `<h2>` of this section and `Enjoy a cocktail at our stylish bar, or savor a fine dining experience with a backdrop of stunning ocean views.` should be the `<p>` since the HTML should make sense semantically without any visuals.